torch/csrc/Module.cpp

```diff
@@ -299,6 +299,13 @@ PyObject *THPModule_hasDistributed(PyObject *_unused)
 #endif
 }
 
+static PyObject *THPModule_detailedVersion(PyObject *module)
```

Comment:
nit: shall we start using pybind universally?

Reply:
For things like this one sure, but I don't trust pybind enough to feel ok with using it for our performance-critical eager functions.